### ISSUE TITLE
frontend: x11 grabs keyboard focus, unable to type into flyout input boxes

### ISSUE THREAD

**Issue by haraldschilly** [sagemathinc/cocalc#6878]
I noticed that in at least some cases, X11 grabs the keyboard focus and it's no longer possible to type into the filter input boxes in the files flyout. There was a similar bug with the side chat – not sure how this was solved. In any case, this one feels similar. 

workaround (sort of): close the x11 editor tab
